### PR TITLE
Load j2objcHome from local.properties or environment variable; Fixes …

### PR DESCRIPTION
**Pull request by advayDev1**
…#106.

### CODE REVIEW

File: src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy [hunk 1]
```diff
@@ -86,17 +86,28 @@ class J2objcUtils {
...
     // @Input getJ2ObjCHome() method can be used freely inside the task action
     static def j2objcHome(Project proj) {
-        def result = proj.j2objcConfig.j2objcHome
+        def localPropertiesFile = new File(proj.rootDir, 'local.properties')
+        String result = null
+        if (localPropertiesFile.exists()) {
+            Properties localProperties = new Properties();
+            localPropertiesFile.withInputStream {
+                localProperties.load it
+            }
```
Comment (brunobowden) [inline src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy]: I found this to be a clearer way of loading the properties file:

    Properties localProperties = new Properties();
    localProperties.load(new FileInputStream(localPropertiesFile))

File: src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy [hunk 2]
```diff
@@ -86,17 +86,28 @@ class J2objcUtils {
...
+            result = localProperties.getProperty('j2objc.home')
```
Comment (brunobowden) [inline src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy]: You've confirmed that if `j2objc.home` doesn't exist then getProperty returns null?

Comment (advayDev1) [inline src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy]: javaDoc says so.

File: src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy (hunk 1 above)
Comment (advayDev1) [inline src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy]: no, that leaves the File descriptor open.  withInputStream ensures things get closed properly.

Comment (brunobowden) [inline src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcUtils.groovy]: I'll fix the prefixes.properties loading to match.

**Pull request merged**